recipes/seqan3/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
Please add the `cmake_find_package` generator +
 use `seqan3::seqan3` in the test package CMake script.

https://github.com/seqan/seqan3/blob/24a24419d2e8e7b936c8962c63c5c514017059ed/build_system/seqan3-config.cmake#L498-L511

Reply:
https://github.com/conan-io/conan-center-index/pull/10924